package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "coc-explorer",
-  "version": "0.11.1",
+  "version": "0.11.2",
```

No need to modify the version, because the publishing program will automatically update the version
```suggestion
  "version": "0.11.1",
```